examples/Cargo.toml

```diff
@@ -20,6 +20,7 @@ uuid = "1.0"
 tower = "0.4"
 stats_alloc = "0.1"
 clap = { version = "3.2.4", features = ["derive"] }
+env_logger = "0.10"
```

Comment:
For the future clarity, let's add a comment saying that this dependency is currently used only by the Book (as one might wonder why it's put there, having checked that `cargo check --all-targets` succeeds even with that dependency removed).

Or, even better, I suggest visually extracting a section in `examples/Cargo.toml` that lists Book-only deps.

Reply:
From a higher perspective, this possible confusion arises because the Book is actually a secondary project that is configured by the same `Cargo.toml` as the primary one (`examples` set of crates).